app/models/miq_server/worker_management/monitor.rb

```diff
@@ -147,8 +147,22 @@ def sync_needed?
       start_apache               if roles_changed &&  apache_needed?
 
       reset_queue_messages       if config_changed || roles_changed
+
+      update_sync_timestamp(@last_sync)
```

Comment:
I guess this isn't really a "message to be processed", and just a single key/val... so maybe not so relevant.